apk-generator/v1/node/main.js

```diff
@@ -1,87 +1,67 @@
-                $('#jsonupload-input').hide(100);
-                $('#eventapi-input').hide(100);
+            if ($(this).val() === "mockjson") {
+                $("#jsonupload-input").hide(100);
+                $("#eventapi-input").hide(100);
             }
 
-            if ($(this).val() === 'jsonupload') {
-                $('#jsonupload-input').show(100);
-                $('#eventapi-input').hide(100);
+            if ($(this).val() === "jsonupload") {
+                $("#jsonupload-input").show(100);
+                $("#eventapi-input").hide(100);
             }
 
-            if ($(this).val() === 'eventapi') {
-                $('#eventapi-input').show(100);
-                $('#jsonupload-input').hide(100);
+            if ($(this).val() === "eventapi") {
+                $("#eventapi-input").show(100);
+                $("#jsonupload-input").hide(100);
             }
         }
     });
-var $ = jQuery;
-var timestamp = Number(new Date());
-var form = document.querySelector("form");
-var config = {
-apiKey: "AIzaSyC_h1ca4ntwFr3j1Y56OwvTEj1geq3I-qQ",
-authDomain: "fossasia-a729e.firebaseapp.com",
-databaseURL: "https://fossasia-a729e.firebaseio.com",
-storageBucket: "fossasia-a729e.appspot.com",
-};
 
 firebase.initializeApp(config);
 var database = firebase.database();
-database.ref('users/1471606817051').on('value',function(snap) {
-    console.log("enna")
-    console.log(snap.val())
+database.ref("users/1471606817051").on("value", function (snap) {
+    console.log("enna");
+    console.log(snap.val());
 })
-form.addEventListener("submit", function(event) {
+form.addEventListener("submit", function (event) {
     event.preventDefault();
-    $('.progress').css('display', 'block');
-    $('#generator-progress').css('display', 'block')
+    $(".progress").css("display", "block");
+    $("#generator-progress").css("display", "block");
     var ary = $(form).serializeArray();
     var obj = {};
     for (var a = 0; a < ary.length; a++) obj[ary[a].name] = ary[a].value;
     console.log("JSON", obj);
-    if (obj.Email == "" || obj.App_Name == "") {
+    if (obj.Email === "" || obj.App_Name === "") {
         alert("It seems like you forgot to fill up your email address or app's name");
         setTimeout("location.reload(true);", 1);
     } else {
         alert("Please wait while we generate the app, meanwhile you can stick around to directly download it.The app will also be emailed to you.");
-        var file_data = $('#uploadZip').prop('files')[0];
-        var form_data = new FormData();
-        form_data.append('file', file_data);
-        firebase.database().ref('users/' + timestamp).set(obj);
-        database.ref('users/' + timestamp).once('value').then(function(snapshot) {
+        var fileData = $("#uploadZip").prop("files")[0];
+        var formData = new FormData();
+        formData.append("file", fileData);
+        firebase.database().ref("users/" + timestamp).set(obj);
```

Comment:
I think firebase is initialised before i don't know why codacy showing issue. I have just change single-quoted-string to double-quoted-string